view_all_inc.php

```diff
@@ -111,10 +111,15 @@
 		<div class="btn-toolbar">
 			<div class="btn-group pull-left">
 		<?php
-			# -- Print and Export links --
+			# -- Print link --
```

Comment:
may I have to commit the solution based on @atrol preference or wait @vboctor answer?